src/lib.rs

```diff
@@ -63,6 +64,32 @@ impl Default for Params {
     }
 }
 
+fn arcpoints(
+    num_circles: usize,
+    idx: usize,
+    o: f64,
```

Comment:
Robin and I are doing a little code review here on a call, and we don't know what `o` is. Could we rename it to be more clear? Is it `offset`?

Reply:
Just took a look and it's Angular offset:

```rust
    // Angle offset so first segment is North
    let o = std::f64::consts::PI / (num_segments as f64);
```

Reply:
Yes for sure – I just used the existing names.